easybuild/easyconfigs/m/modin/modin-0.32.0-foss-2024a.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'PythonBundle'
+
+name = 'modin'
+version = '0.32.0'
+
+homepage = 'https://github.com/modin-project/modin'
+description = """Modin uses Ray, Dask or Unidist to provide an effortless way to speed up your pandas notebooks,
+scripts, and libraries. """
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+builddependencies = [
+    ('Cython', '3.0.10'),
+    # Needed for tests
+    ('boto3', '1.35.36'),
+    ('s3fs', '2024.9.0'),
+]
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('Python-bundle-PyPI', '2024.06'),
+    ('SciPy-bundle', '2024.05'),
+    ('dask', '2024.9.1'),
+    ('OpenMPI', '5.0.3'),
+    ('Ray-project', '2.37.0'),
+    ('Arrow', '17.0.0'),
+]
+
+use_pip = True
+
+exts_list = [
+    # modin has another backend, "unidist", for MPI, but it seems broken currently.
```

Comment:
can you add a bit more info why it's broken?
what's the error message?
maybe open a github issue that you can link to?

Reply:
There is no error message, trying to build it and running tests with unidist as the backend, or even just importing it and trying to use anything, e.g. `modin.pandas.DataFrame()`, results in all your nodes running some kind of unidist init code and firing up all the nodes to 50%. I could not find any documentation of this behaviour.

I have tried the usual remedies to attempt to debug it but I can't get it to work at all.

Reply:
Updated to add a little more info, but there is really not much that I can say about it.

Reply:
@smoors I think it's fixed.

Reply:
great, thanks a lot!